contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -116,6 +127,9 @@ class AzureLogHandler(TransportMixin, BaseLogHandler):
     def __init__(self, **options):
         self.options = Options(**options)
         utils.validate_instrumentation_key(self.options.instrumentation_key)
+        if self.options.logging_sampling_rate < 0 or \
```

Comment:
nit:You can move this at the beginning of the method to avoid doing anything else if sampling rate is incorrect

Reply:
Validating the instrumentation key is also a validation and fast fail in itself.